src/explore-education-statistics-admin/src/pages/release/data/components/DataFileDetailsTable.tsx

```diff
@@ -113,33 +113,27 @@ const DataFileDetailsTable = ({
         <tr>
           <th scope="row">Data file size</th>
           <td data-testid="Data file size">
-            {!dataFile.isQueuedZipUpload && (
-              <>
-                {dataFile.fileSize.size.toLocaleString()}{' '}
-                {dataFile.fileSize.unit}
-              </>
-            )}
+            <>
+              {dataFile.fileSize.size.toLocaleString()} {dataFile.fileSize.unit}
```

Comment:
Would suggest making this a single template literal string:
```tsx
{`${dataFile.fileSize.size.toLocaleString()} ${dataFile.fileSize.unit}`}
```

It's a not a big difference, but if we don't do this, it can make it harder for tests to find specific pieces of text (particularly the Robot tests).

Reply:
Thanks, made this change